### ISSUE TITLE
port GRBL 1.1f for STM32

### ISSUE THREAD

**Issue by mkeyno** [MaxwellXyao/GRBL-0.9j-for-STM32#1]
hi, have any plan to port the last updated GRBL1.1f on [STM 32](http://www.rogerclark.net/stm32f103-and-maple-maple-mini-with-arduino-1-5-x-ide/)

**Comment by MaxwellXyao** [comment]
Yes, I have such a plan, after I have studied the current version(=・ω・=)

**Comment by mkeyno** [comment]
@MaxwellXyao  would you mind check this  issue https://github.com/gnea/grbl/issues/197
